// cssstyle/lib/properties.js
'use strict';

// autogenerated

/*
 *
 * http://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSS2Properties
 */

module.exports = function (prototype) {
    Object.defineProperty(prototype, 'alignmentBaseline', {
        get: function () {
            var definition = require('./properties/alignmentBaseline').definition;
            Object.defineProperty(prototype, 'alignmentBaseline', definition);
            return this.alignmentBaseline;
        },
        set: function (v) {
            var definition = require('./properties/alignmentBaseline').definition;
            Object.defineProperty(prototype, 'alignmentBaseline', definition);
            this.alignmentBaseline = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'alignment-baseline', {
        get: function () {
            var definition = require('./properties/alignmentBaseline').definition;
            Object.defineProperty(prototype, 'alignmentBaseline', definition);
            return this.alignmentBaseline;
        },
        set: function (v) {
            var definition = require('./properties/alignmentBaseline').definition;
            Object.defineProperty(prototype, 'alignmentBaseline', definition);
            this.alignmentBaseline = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'azimuth', {
        get: function () {
            var definition = require('./properties/azimuth').definition;
            Object.defineProperty(prototype, 'azimuth', definition);
            return this.azimuth;
        },
        set: function (v) {
            var definition = require('./properties/azimuth').definition;
            Object.defineProperty(prototype, 'azimuth', definition);
            this.azimuth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background', {
        get: function () {
            var definition = require('./properties/background').definition;
            Object.defineProperty(prototype, 'background', definition);
            return this.background;
        },
        set: function (v) {
            var definition = require('./properties/background').definition;
            Object.defineProperty(prototype, 'background', definition);
            this.background = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundAttachment', {
        get: function () {
            var definition = require('./properties/backgroundAttachment').definition;
            Object.defineProperty(prototype, 'backgroundAttachment', definition);
            return this.backgroundAttachment;
        },
        set: function (v) {
            var definition = require('./properties/backgroundAttachment').definition;
            Object.defineProperty(prototype, 'backgroundAttachment', definition);
            this.backgroundAttachment = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-attachment', {
        get: function () {
            var definition = require('./properties/backgroundAttachment').definition;
            Object.defineProperty(prototype, 'backgroundAttachment', definition);
            return this.backgroundAttachment;
        },
        set: function (v) {
            var definition = require('./properties/backgroundAttachment').definition;
            Object.defineProperty(prototype, 'backgroundAttachment', definition);
            this.backgroundAttachment = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundClip', {
        get: function () {
            var definition = require('./properties/backgroundClip').definition;
            Object.defineProperty(prototype, 'backgroundClip', definition);
            return this.backgroundClip;
        },
        set: function (v) {
            var definition = require('./properties/backgroundClip').definition;
            Object.defineProperty(prototype, 'backgroundClip', definition);
            this.backgroundClip = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-clip', {
        get: function () {
            var definition = require('./properties/backgroundClip').definition;
            Object.defineProperty(prototype, 'backgroundClip', definition);
            return this.backgroundClip;
        },
        set: function (v) {
            var definition = require('./properties/backgroundClip').definition;
            Object.defineProperty(prototype, 'backgroundClip', definition);
            this.backgroundClip = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundColor', {
        get: function () {
            var definition = require('./properties/backgroundColor').definition;
            Object.defineProperty(prototype, 'backgroundColor', definition);
            return this.backgroundColor;
        },
        set: function (v) {
            var definition = require('./properties/backgroundColor').definition;
            Object.defineProperty(prototype, 'backgroundColor', definition);
            this.backgroundColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-color', {
        get: function () {
            var definition = require('./properties/backgroundColor').definition;
            Object.defineProperty(prototype, 'backgroundColor', definition);
            return this.backgroundColor;
        },
        set: function (v) {
            var definition = require('./properties/backgroundColor').definition;
            Object.defineProperty(prototype, 'backgroundColor', definition);
            this.backgroundColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundImage', {
        get: function () {
            var definition = require('./properties/backgroundImage').definition;
            Object.defineProperty(prototype, 'backgroundImage', definition);
            return this.backgroundImage;
        },
        set: function (v) {
            var definition = require('./properties/backgroundImage').definition;
            Object.defineProperty(prototype, 'backgroundImage', definition);
            this.backgroundImage = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-image', {
        get: function () {
            var definition = require('./properties/backgroundImage').definition;
            Object.defineProperty(prototype, 'backgroundImage', definition);
            return this.backgroundImage;
        },
        set: function (v) {
            var definition = require('./properties/backgroundImage').definition;
            Object.defineProperty(prototype, 'backgroundImage', definition);
            this.backgroundImage = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundOrigin', {
        get: function () {
            var definition = require('./properties/backgroundOrigin').definition;
            Object.defineProperty(prototype, 'backgroundOrigin', definition);
            return this.backgroundOrigin;
        },
        set: function (v) {
            var definition = require('./properties/backgroundOrigin').definition;
            Object.defineProperty(prototype, 'backgroundOrigin', definition);
            this.backgroundOrigin = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-origin', {
        get: function () {
            var definition = require('./properties/backgroundOrigin').definition;
            Object.defineProperty(prototype, 'backgroundOrigin', definition);
            return this.backgroundOrigin;
        },
        set: function (v) {
            var definition = require('./properties/backgroundOrigin').definition;
            Object.defineProperty(prototype, 'backgroundOrigin', definition);
            this.backgroundOrigin = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundPosition', {
        get: function () {
            var definition = require('./properties/backgroundPosition').definition;
            Object.defineProperty(prototype, 'backgroundPosition', definition);
            return this.backgroundPosition;
        },
        set: function (v) {
            var definition = require('./properties/backgroundPosition').definition;
            Object.defineProperty(prototype, 'backgroundPosition', definition);
            this.backgroundPosition = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-position', {
        get: function () {
            var definition = require('./properties/backgroundPosition').definition;
            Object.defineProperty(prototype, 'backgroundPosition', definition);
            return this.backgroundPosition;
        },
        set: function (v) {
            var definition = require('./properties/backgroundPosition').definition;
            Object.defineProperty(prototype, 'backgroundPosition', definition);
            this.backgroundPosition = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundPositionX', {
        get: function () {
            var definition = require('./properties/backgroundPositionX').definition;
            Object.defineProperty(prototype, 'backgroundPositionX', definition);
            return this.backgroundPositionX;
        },
        set: function (v) {
            var definition = require('./properties/backgroundPositionX').definition;
            Object.defineProperty(prototype, 'backgroundPositionX', definition);
            this.backgroundPositionX = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-position-x', {
        get: function () {
            var definition = require('./properties/backgroundPositionX').definition;
            Object.defineProperty(prototype, 'backgroundPositionX', definition);
            return this.backgroundPositionX;
        },
        set: function (v) {
            var definition = require('./properties/backgroundPositionX').definition;
            Object.defineProperty(prototype, 'backgroundPositionX', definition);
            this.backgroundPositionX = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundPositionY', {
        get: function () {
            var definition = require('./properties/backgroundPositionY').definition;
            Object.defineProperty(prototype, 'backgroundPositionY', definition);
            return this.backgroundPositionY;
        },
        set: function (v) {
            var definition = require('./properties/backgroundPositionY').definition;
            Object.defineProperty(prototype, 'backgroundPositionY', definition);
            this.backgroundPositionY = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-position-y', {
        get: function () {
            var definition = require('./properties/backgroundPositionY').definition;
            Object.defineProperty(prototype, 'backgroundPositionY', definition);
            return this.backgroundPositionY;
        },
        set: function (v) {
            var definition = require('./properties/backgroundPositionY').definition;
            Object.defineProperty(prototype, 'backgroundPositionY', definition);
            this.backgroundPositionY = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundRepeat', {
        get: function () {
            var definition = require('./properties/backgroundRepeat').definition;
            Object.defineProperty(prototype, 'backgroundRepeat', definition);
            return this.backgroundRepeat;
        },
        set: function (v) {
            var definition = require('./properties/backgroundRepeat').definition;
            Object.defineProperty(prototype, 'backgroundRepeat', definition);
            this.backgroundRepeat = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-repeat', {
        get: function () {
            var definition = require('./properties/backgroundRepeat').definition;
            Object.defineProperty(prototype, 'backgroundRepeat', definition);
            return this.backgroundRepeat;
        },
        set: function (v) {
            var definition = require('./properties/backgroundRepeat').definition;
            Object.defineProperty(prototype, 'backgroundRepeat', definition);
            this.backgroundRepeat = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundRepeatX', {
        get: function () {
            var definition = require('./properties/backgroundRepeatX').definition;
            Object.defineProperty(prototype, 'backgroundRepeatX', definition);
            return this.backgroundRepeatX;
        },
        set: function (v) {
            var definition = require('./properties/backgroundRepeatX').definition;
            Object.defineProperty(prototype, 'backgroundRepeatX', definition);
            this.backgroundRepeatX = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-repeat-x', {
        get: function () {
            var definition = require('./properties/backgroundRepeatX').definition;
            Object.defineProperty(prototype, 'backgroundRepeatX', definition);
            return this.backgroundRepeatX;
        },
        set: function (v) {
            var definition = require('./properties/backgroundRepeatX').definition;
            Object.defineProperty(prototype, 'backgroundRepeatX', definition);
            this.backgroundRepeatX = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundRepeatY', {
        get: function () {
            var definition = require('./properties/backgroundRepeatY').definition;
            Object.defineProperty(prototype, 'backgroundRepeatY', definition);
            return this.backgroundRepeatY;
        },
        set: function (v) {
            var definition = require('./properties/backgroundRepeatY').definition;
            Object.defineProperty(prototype, 'backgroundRepeatY', definition);
            this.backgroundRepeatY = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-repeat-y', {
        get: function () {
            var definition = require('./properties/backgroundRepeatY').definition;
            Object.defineProperty(prototype, 'backgroundRepeatY', definition);
            return this.backgroundRepeatY;
        },
        set: function (v) {
            var definition = require('./properties/backgroundRepeatY').definition;
            Object.defineProperty(prototype, 'backgroundRepeatY', definition);
            this.backgroundRepeatY = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'backgroundSize', {
        get: function () {
            var definition = require('./properties/backgroundSize').definition;
            Object.defineProperty(prototype, 'backgroundSize', definition);
            return this.backgroundSize;
        },
        set: function (v) {
            var definition = require('./properties/backgroundSize').definition;
            Object.defineProperty(prototype, 'backgroundSize', definition);
            this.backgroundSize = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'background-size', {
        get: function () {
            var definition = require('./properties/backgroundSize').definition;
            Object.defineProperty(prototype, 'backgroundSize', definition);
            return this.backgroundSize;
        },
        set: function (v) {
            var definition = require('./properties/backgroundSize').definition;
            Object.defineProperty(prototype, 'backgroundSize', definition);
            this.backgroundSize = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'baselineShift', {
        get: function () {
            var definition = require('./properties/baselineShift').definition;
            Object.defineProperty(prototype, 'baselineShift', definition);
            return this.baselineShift;
        },
        set: function (v) {
            var definition = require('./properties/baselineShift').definition;
            Object.defineProperty(prototype, 'baselineShift', definition);
            this.baselineShift = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'baseline-shift', {
        get: function () {
            var definition = require('./properties/baselineShift').definition;
            Object.defineProperty(prototype, 'baselineShift', definition);
            return this.baselineShift;
        },
        set: function (v) {
            var definition = require('./properties/baselineShift').definition;
            Object.defineProperty(prototype, 'baselineShift', definition);
            this.baselineShift = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border', {
        get: function () {
            var definition = require('./properties/border').definition;
            Object.defineProperty(prototype, 'border', definition);
            return this.border;
        },
        set: function (v) {
            var definition = require('./properties/border').definition;
            Object.defineProperty(prototype, 'border', definition);
            this.border = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderBottom', {
        get: function () {
            var definition = require('./properties/borderBottom').definition;
            Object.defineProperty(prototype, 'borderBottom', definition);
            return this.borderBottom;
        },
        set: function (v) {
            var definition = require('./properties/borderBottom').definition;
            Object.defineProperty(prototype, 'borderBottom', definition);
            this.borderBottom = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-bottom', {
        get: function () {
            var definition = require('./properties/borderBottom').definition;
            Object.defineProperty(prototype, 'borderBottom', definition);
            return this.borderBottom;
        },
        set: function (v) {
            var definition = require('./properties/borderBottom').definition;
            Object.defineProperty(prototype, 'borderBottom', definition);
            this.borderBottom = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderBottomColor', {
        get: function () {
            var definition = require('./properties/borderBottomColor').definition;
            Object.defineProperty(prototype, 'borderBottomColor', definition);
            return this.borderBottomColor;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomColor').definition;
            Object.defineProperty(prototype, 'borderBottomColor', definition);
            this.borderBottomColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-bottom-color', {
        get: function () {
            var definition = require('./properties/borderBottomColor').definition;
            Object.defineProperty(prototype, 'borderBottomColor', definition);
            return this.borderBottomColor;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomColor').definition;
            Object.defineProperty(prototype, 'borderBottomColor', definition);
            this.borderBottomColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderBottomLeftRadius', {
        get: function () {
            var definition = require('./properties/borderBottomLeftRadius').definition;
            Object.defineProperty(prototype, 'borderBottomLeftRadius', definition);
            return this.borderBottomLeftRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomLeftRadius').definition;
            Object.defineProperty(prototype, 'borderBottomLeftRadius', definition);
            this.borderBottomLeftRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-bottom-left-radius', {
        get: function () {
            var definition = require('./properties/borderBottomLeftRadius').definition;
            Object.defineProperty(prototype, 'borderBottomLeftRadius', definition);
            return this.borderBottomLeftRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomLeftRadius').definition;
            Object.defineProperty(prototype, 'borderBottomLeftRadius', definition);
            this.borderBottomLeftRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderBottomRightRadius', {
        get: function () {
            var definition = require('./properties/borderBottomRightRadius').definition;
            Object.defineProperty(prototype, 'borderBottomRightRadius', definition);
            return this.borderBottomRightRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomRightRadius').definition;
            Object.defineProperty(prototype, 'borderBottomRightRadius', definition);
            this.borderBottomRightRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-bottom-right-radius', {
        get: function () {
            var definition = require('./properties/borderBottomRightRadius').definition;
            Object.defineProperty(prototype, 'borderBottomRightRadius', definition);
            return this.borderBottomRightRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomRightRadius').definition;
            Object.defineProperty(prototype, 'borderBottomRightRadius', definition);
            this.borderBottomRightRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderBottomStyle', {
        get: function () {
            var definition = require('./properties/borderBottomStyle').definition;
            Object.defineProperty(prototype, 'borderBottomStyle', definition);
            return this.borderBottomStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomStyle').definition;
            Object.defineProperty(prototype, 'borderBottomStyle', definition);
            this.borderBottomStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-bottom-style', {
        get: function () {
            var definition = require('./properties/borderBottomStyle').definition;
            Object.defineProperty(prototype, 'borderBottomStyle', definition);
            return this.borderBottomStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomStyle').definition;
            Object.defineProperty(prototype, 'borderBottomStyle', definition);
            this.borderBottomStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderBottomWidth', {
        get: function () {
            var definition = require('./properties/borderBottomWidth').definition;
            Object.defineProperty(prototype, 'borderBottomWidth', definition);
            return this.borderBottomWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomWidth').definition;
            Object.defineProperty(prototype, 'borderBottomWidth', definition);
            this.borderBottomWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-bottom-width', {
        get: function () {
            var definition = require('./properties/borderBottomWidth').definition;
            Object.defineProperty(prototype, 'borderBottomWidth', definition);
            return this.borderBottomWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderBottomWidth').definition;
            Object.defineProperty(prototype, 'borderBottomWidth', definition);
            this.borderBottomWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderCollapse', {
        get: function () {
            var definition = require('./properties/borderCollapse').definition;
            Object.defineProperty(prototype, 'borderCollapse', definition);
            return this.borderCollapse;
        },
        set: function (v) {
            var definition = require('./properties/borderCollapse').definition;
            Object.defineProperty(prototype, 'borderCollapse', definition);
            this.borderCollapse = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-collapse', {
        get: function () {
            var definition = require('./properties/borderCollapse').definition;
            Object.defineProperty(prototype, 'borderCollapse', definition);
            return this.borderCollapse;
        },
        set: function (v) {
            var definition = require('./properties/borderCollapse').definition;
            Object.defineProperty(prototype, 'borderCollapse', definition);
            this.borderCollapse = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderColor', {
        get: function () {
            var definition = require('./properties/borderColor').definition;
            Object.defineProperty(prototype, 'borderColor', definition);
            return this.borderColor;
        },
        set: function (v) {
            var definition = require('./properties/borderColor').definition;
            Object.defineProperty(prototype, 'borderColor', definition);
            this.borderColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-color', {
        get: function () {
            var definition = require('./properties/borderColor').definition;
            Object.defineProperty(prototype, 'borderColor', definition);
            return this.borderColor;
        },
        set: function (v) {
            var definition = require('./properties/borderColor').definition;
            Object.defineProperty(prototype, 'borderColor', definition);
            this.borderColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderImage', {
        get: function () {
            var definition = require('./properties/borderImage').definition;
            Object.defineProperty(prototype, 'borderImage', definition);
            return this.borderImage;
        },
        set: function (v) {
            var definition = require('./properties/borderImage').definition;
            Object.defineProperty(prototype, 'borderImage', definition);
            this.borderImage = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-image', {
        get: function () {
            var definition = require('./properties/borderImage').definition;
            Object.defineProperty(prototype, 'borderImage', definition);
            return this.borderImage;
        },
        set: function (v) {
            var definition = require('./properties/borderImage').definition;
            Object.defineProperty(prototype, 'borderImage', definition);
            this.borderImage = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderImageOutset', {
        get: function () {
            var definition = require('./properties/borderImageOutset').definition;
            Object.defineProperty(prototype, 'borderImageOutset', definition);
            return this.borderImageOutset;
        },
        set: function (v) {
            var definition = require('./properties/borderImageOutset').definition;
            Object.defineProperty(prototype, 'borderImageOutset', definition);
            this.borderImageOutset = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-image-outset', {
        get: function () {
            var definition = require('./properties/borderImageOutset').definition;
            Object.defineProperty(prototype, 'borderImageOutset', definition);
            return this.borderImageOutset;
        },
        set: function (v) {
            var definition = require('./properties/borderImageOutset').definition;
            Object.defineProperty(prototype, 'borderImageOutset', definition);
            this.borderImageOutset = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderImageRepeat', {
        get: function () {
            var definition = require('./properties/borderImageRepeat').definition;
            Object.defineProperty(prototype, 'borderImageRepeat', definition);
            return this.borderImageRepeat;
        },
        set: function (v) {
            var definition = require('./properties/borderImageRepeat').definition;
            Object.defineProperty(prototype, 'borderImageRepeat', definition);
            this.borderImageRepeat = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-image-repeat', {
        get: function () {
            var definition = require('./properties/borderImageRepeat').definition;
            Object.defineProperty(prototype, 'borderImageRepeat', definition);
            return this.borderImageRepeat;
        },
        set: function (v) {
            var definition = require('./properties/borderImageRepeat').definition;
            Object.defineProperty(prototype, 'borderImageRepeat', definition);
            this.borderImageRepeat = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderImageSlice', {
        get: function () {
            var definition = require('./properties/borderImageSlice').definition;
            Object.defineProperty(prototype, 'borderImageSlice', definition);
            return this.borderImageSlice;
        },
        set: function (v) {
            var definition = require('./properties/borderImageSlice').definition;
            Object.defineProperty(prototype, 'borderImageSlice', definition);
            this.borderImageSlice = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-image-slice', {
        get: function () {
            var definition = require('./properties/borderImageSlice').definition;
            Object.defineProperty(prototype, 'borderImageSlice', definition);
            return this.borderImageSlice;
        },
        set: function (v) {
            var definition = require('./properties/borderImageSlice').definition;
            Object.defineProperty(prototype, 'borderImageSlice', definition);
            this.borderImageSlice = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderImageSource', {
        get: function () {
            var definition = require('./properties/borderImageSource').definition;
            Object.defineProperty(prototype, 'borderImageSource', definition);
            return this.borderImageSource;
        },
        set: function (v) {
            var definition = require('./properties/borderImageSource').definition;
            Object.defineProperty(prototype, 'borderImageSource', definition);
            this.borderImageSource = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-image-source', {
        get: function () {
            var definition = require('./properties/borderImageSource').definition;
            Object.defineProperty(prototype, 'borderImageSource', definition);
            return this.borderImageSource;
        },
        set: function (v) {
            var definition = require('./properties/borderImageSource').definition;
            Object.defineProperty(prototype, 'borderImageSource', definition);
            this.borderImageSource = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderImageWidth', {
        get: function () {
            var definition = require('./properties/borderImageWidth').definition;
            Object.defineProperty(prototype, 'borderImageWidth', definition);
            return this.borderImageWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderImageWidth').definition;
            Object.defineProperty(prototype, 'borderImageWidth', definition);
            this.borderImageWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-image-width', {
        get: function () {
            var definition = require('./properties/borderImageWidth').definition;
            Object.defineProperty(prototype, 'borderImageWidth', definition);
            return this.borderImageWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderImageWidth').definition;
            Object.defineProperty(prototype, 'borderImageWidth', definition);
            this.borderImageWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderLeft', {
        get: function () {
            var definition = require('./properties/borderLeft').definition;
            Object.defineProperty(prototype, 'borderLeft', definition);
            return this.borderLeft;
        },
        set: function (v) {
            var definition = require('./properties/borderLeft').definition;
            Object.defineProperty(prototype, 'borderLeft', definition);
            this.borderLeft = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-left', {
        get: function () {
            var definition = require('./properties/borderLeft').definition;
            Object.defineProperty(prototype, 'borderLeft', definition);
            return this.borderLeft;
        },
        set: function (v) {
            var definition = require('./properties/borderLeft').definition;
            Object.defineProperty(prototype, 'borderLeft', definition);
            this.borderLeft = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderLeftColor', {
        get: function () {
            var definition = require('./properties/borderLeftColor').definition;
            Object.defineProperty(prototype, 'borderLeftColor', definition);
            return this.borderLeftColor;
        },
        set: function (v) {
            var definition = require('./properties/borderLeftColor').definition;
            Object.defineProperty(prototype, 'borderLeftColor', definition);
            this.borderLeftColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-left-color', {
        get: function () {
            var definition = require('./properties/borderLeftColor').definition;
            Object.defineProperty(prototype, 'borderLeftColor', definition);
            return this.borderLeftColor;
        },
        set: function (v) {
            var definition = require('./properties/borderLeftColor').definition;
            Object.defineProperty(prototype, 'borderLeftColor', definition);
            this.borderLeftColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderLeftStyle', {
        get: function () {
            var definition = require('./properties/borderLeftStyle').definition;
            Object.defineProperty(prototype, 'borderLeftStyle', definition);
            return this.borderLeftStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderLeftStyle').definition;
            Object.defineProperty(prototype, 'borderLeftStyle', definition);
            this.borderLeftStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-left-style', {
        get: function () {
            var definition = require('./properties/borderLeftStyle').definition;
            Object.defineProperty(prototype, 'borderLeftStyle', definition);
            return this.borderLeftStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderLeftStyle').definition;
            Object.defineProperty(prototype, 'borderLeftStyle', definition);
            this.borderLeftStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderLeftWidth', {
        get: function () {
            var definition = require('./properties/borderLeftWidth').definition;
            Object.defineProperty(prototype, 'borderLeftWidth', definition);
            return this.borderLeftWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderLeftWidth').definition;
            Object.defineProperty(prototype, 'borderLeftWidth', definition);
            this.borderLeftWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-left-width', {
        get: function () {
            var definition = require('./properties/borderLeftWidth').definition;
            Object.defineProperty(prototype, 'borderLeftWidth', definition);
            return this.borderLeftWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderLeftWidth').definition;
            Object.defineProperty(prototype, 'borderLeftWidth', definition);
            this.borderLeftWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderRadius', {
        get: function () {
            var definition = require('./properties/borderRadius').definition;
            Object.defineProperty(prototype, 'borderRadius', definition);
            return this.borderRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderRadius').definition;
            Object.defineProperty(prototype, 'borderRadius', definition);
            this.borderRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-radius', {
        get: function () {
            var definition = require('./properties/borderRadius').definition;
            Object.defineProperty(prototype, 'borderRadius', definition);
            return this.borderRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderRadius').definition;
            Object.defineProperty(prototype, 'borderRadius', definition);
            this.borderRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderRight', {
        get: function () {
            var definition = require('./properties/borderRight').definition;
            Object.defineProperty(prototype, 'borderRight', definition);
            return this.borderRight;
        },
        set: function (v) {
            var definition = require('./properties/borderRight').definition;
            Object.defineProperty(prototype, 'borderRight', definition);
            this.borderRight = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-right', {
        get: function () {
            var definition = require('./properties/borderRight').definition;
            Object.defineProperty(prototype, 'borderRight', definition);
            return this.borderRight;
        },
        set: function (v) {
            var definition = require('./properties/borderRight').definition;
            Object.defineProperty(prototype, 'borderRight', definition);
            this.borderRight = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderRightColor', {
        get: function () {
            var definition = require('./properties/borderRightColor').definition;
            Object.defineProperty(prototype, 'borderRightColor', definition);
            return this.borderRightColor;
        },
        set: function (v) {
            var definition = require('./properties/borderRightColor').definition;
            Object.defineProperty(prototype, 'borderRightColor', definition);
            this.borderRightColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-right-color', {
        get: function () {
            var definition = require('./properties/borderRightColor').definition;
            Object.defineProperty(prototype, 'borderRightColor', definition);
            return this.borderRightColor;
        },
        set: function (v) {
            var definition = require('./properties/borderRightColor').definition;
            Object.defineProperty(prototype, 'borderRightColor', definition);
            this.borderRightColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderRightStyle', {
        get: function () {
            var definition = require('./properties/borderRightStyle').definition;
            Object.defineProperty(prototype, 'borderRightStyle', definition);
            return this.borderRightStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderRightStyle').definition;
            Object.defineProperty(prototype, 'borderRightStyle', definition);
            this.borderRightStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-right-style', {
        get: function () {
            var definition = require('./properties/borderRightStyle').definition;
            Object.defineProperty(prototype, 'borderRightStyle', definition);
            return this.borderRightStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderRightStyle').definition;
            Object.defineProperty(prototype, 'borderRightStyle', definition);
            this.borderRightStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderRightWidth', {
        get: function () {
            var definition = require('./properties/borderRightWidth').definition;
            Object.defineProperty(prototype, 'borderRightWidth', definition);
            return this.borderRightWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderRightWidth').definition;
            Object.defineProperty(prototype, 'borderRightWidth', definition);
            this.borderRightWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-right-width', {
        get: function () {
            var definition = require('./properties/borderRightWidth').definition;
            Object.defineProperty(prototype, 'borderRightWidth', definition);
            return this.borderRightWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderRightWidth').definition;
            Object.defineProperty(prototype, 'borderRightWidth', definition);
            this.borderRightWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderSpacing', {
        get: function () {
            var definition = require('./properties/borderSpacing').definition;
            Object.defineProperty(prototype, 'borderSpacing', definition);
            return this.borderSpacing;
        },
        set: function (v) {
            var definition = require('./properties/borderSpacing').definition;
            Object.defineProperty(prototype, 'borderSpacing', definition);
            this.borderSpacing = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-spacing', {
        get: function () {
            var definition = require('./properties/borderSpacing').definition;
            Object.defineProperty(prototype, 'borderSpacing', definition);
            return this.borderSpacing;
        },
        set: function (v) {
            var definition = require('./properties/borderSpacing').definition;
            Object.defineProperty(prototype, 'borderSpacing', definition);
            this.borderSpacing = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderStyle', {
        get: function () {
            var definition = require('./properties/borderStyle').definition;
            Object.defineProperty(prototype, 'borderStyle', definition);
            return this.borderStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderStyle').definition;
            Object.defineProperty(prototype, 'borderStyle', definition);
            this.borderStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-style', {
        get: function () {
            var definition = require('./properties/borderStyle').definition;
            Object.defineProperty(prototype, 'borderStyle', definition);
            return this.borderStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderStyle').definition;
            Object.defineProperty(prototype, 'borderStyle', definition);
            this.borderStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderTop', {
        get: function () {
            var definition = require('./properties/borderTop').definition;
            Object.defineProperty(prototype, 'borderTop', definition);
            return this.borderTop;
        },
        set: function (v) {
            var definition = require('./properties/borderTop').definition;
            Object.defineProperty(prototype, 'borderTop', definition);
            this.borderTop = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-top', {
        get: function () {
            var definition = require('./properties/borderTop').definition;
            Object.defineProperty(prototype, 'borderTop', definition);
            return this.borderTop;
        },
        set: function (v) {
            var definition = require('./properties/borderTop').definition;
            Object.defineProperty(prototype, 'borderTop', definition);
            this.borderTop = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderTopColor', {
        get: function () {
            var definition = require('./properties/borderTopColor').definition;
            Object.defineProperty(prototype, 'borderTopColor', definition);
            return this.borderTopColor;
        },
        set: function (v) {
            var definition = require('./properties/borderTopColor').definition;
            Object.defineProperty(prototype, 'borderTopColor', definition);
            this.borderTopColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-top-color', {
        get: function () {
            var definition = require('./properties/borderTopColor').definition;
            Object.defineProperty(prototype, 'borderTopColor', definition);
            return this.borderTopColor;
        },
        set: function (v) {
            var definition = require('./properties/borderTopColor').definition;
            Object.defineProperty(prototype, 'borderTopColor', definition);
            this.borderTopColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderTopLeftRadius', {
        get: function () {
            var definition = require('./properties/borderTopLeftRadius').definition;
            Object.defineProperty(prototype, 'borderTopLeftRadius', definition);
            return this.borderTopLeftRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderTopLeftRadius').definition;
            Object.defineProperty(prototype, 'borderTopLeftRadius', definition);
            this.borderTopLeftRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-top-left-radius', {
        get: function () {
            var definition = require('./properties/borderTopLeftRadius').definition;
            Object.defineProperty(prototype, 'borderTopLeftRadius', definition);
            return this.borderTopLeftRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderTopLeftRadius').definition;
            Object.defineProperty(prototype, 'borderTopLeftRadius', definition);
            this.borderTopLeftRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderTopRightRadius', {
        get: function () {
            var definition = require('./properties/borderTopRightRadius').definition;
            Object.defineProperty(prototype, 'borderTopRightRadius', definition);
            return this.borderTopRightRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderTopRightRadius').definition;
            Object.defineProperty(prototype, 'borderTopRightRadius', definition);
            this.borderTopRightRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-top-right-radius', {
        get: function () {
            var definition = require('./properties/borderTopRightRadius').definition;
            Object.defineProperty(prototype, 'borderTopRightRadius', definition);
            return this.borderTopRightRadius;
        },
        set: function (v) {
            var definition = require('./properties/borderTopRightRadius').definition;
            Object.defineProperty(prototype, 'borderTopRightRadius', definition);
            this.borderTopRightRadius = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderTopStyle', {
        get: function () {
            var definition = require('./properties/borderTopStyle').definition;
            Object.defineProperty(prototype, 'borderTopStyle', definition);
            return this.borderTopStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderTopStyle').definition;
            Object.defineProperty(prototype, 'borderTopStyle', definition);
            this.borderTopStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-top-style', {
        get: function () {
            var definition = require('./properties/borderTopStyle').definition;
            Object.defineProperty(prototype, 'borderTopStyle', definition);
            return this.borderTopStyle;
        },
        set: function (v) {
            var definition = require('./properties/borderTopStyle').definition;
            Object.defineProperty(prototype, 'borderTopStyle', definition);
            this.borderTopStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderTopWidth', {
        get: function () {
            var definition = require('./properties/borderTopWidth').definition;
            Object.defineProperty(prototype, 'borderTopWidth', definition);
            return this.borderTopWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderTopWidth').definition;
            Object.defineProperty(prototype, 'borderTopWidth', definition);
            this.borderTopWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-top-width', {
        get: function () {
            var definition = require('./properties/borderTopWidth').definition;
            Object.defineProperty(prototype, 'borderTopWidth', definition);
            return this.borderTopWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderTopWidth').definition;
            Object.defineProperty(prototype, 'borderTopWidth', definition);
            this.borderTopWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'borderWidth', {
        get: function () {
            var definition = require('./properties/borderWidth').definition;
            Object.defineProperty(prototype, 'borderWidth', definition);
            return this.borderWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderWidth').definition;
            Object.defineProperty(prototype, 'borderWidth', definition);
            this.borderWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'border-width', {
        get: function () {
            var definition = require('./properties/borderWidth').definition;
            Object.defineProperty(prototype, 'borderWidth', definition);
            return this.borderWidth;
        },
        set: function (v) {
            var definition = require('./properties/borderWidth').definition;
            Object.defineProperty(prototype, 'borderWidth', definition);
            this.borderWidth = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'bottom', {
        get: function () {
            var definition = require('./properties/bottom').definition;
            Object.defineProperty(prototype, 'bottom', definition);
            return this.bottom;
        },
        set: function (v) {
            var definition = require('./properties/bottom').definition;
            Object.defineProperty(prototype, 'bottom', definition);
            this.bottom = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'boxShadow', {
        get: function () {
            var definition = require('./properties/boxShadow').definition;
            Object.defineProperty(prototype, 'boxShadow', definition);
            return this.boxShadow;
        },
        set: function (v) {
            var definition = require('./properties/boxShadow').definition;
            Object.defineProperty(prototype, 'boxShadow', definition);
            this.boxShadow = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'box-shadow', {
        get: function () {
            var definition = require('./properties/boxShadow').definition;
            Object.defineProperty(prototype, 'boxShadow', definition);
            return this.boxShadow;
        },
        set: function (v) {
            var definition = require('./properties/boxShadow').definition;
            Object.defineProperty(prototype, 'boxShadow', definition);
            this.boxShadow = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'boxSizing', {
        get: function () {
            var definition = require('./properties/boxSizing').definition;
            Object.defineProperty(prototype, 'boxSizing', definition);
            return this.boxSizing;
        },
        set: function (v) {
            var definition = require('./properties/boxSizing').definition;
            Object.defineProperty(prototype, 'boxSizing', definition);
            this.boxSizing = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'box-sizing', {
        get: function () {
            var definition = require('./properties/boxSizing').definition;
            Object.defineProperty(prototype, 'boxSizing', definition);
            return this.boxSizing;
        },
        set: function (v) {
            var definition = require('./properties/boxSizing').definition;
            Object.defineProperty(prototype, 'boxSizing', definition);
            this.boxSizing = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'captionSide', {
        get: function () {
            var definition = require('./properties/captionSide').definition;
            Object.defineProperty(prototype, 'captionSide', definition);
            return this.captionSide;
        },
        set: function (v) {
            var definition = require('./properties/captionSide').definition;
            Object.defineProperty(prototype, 'captionSide', definition);
            this.captionSide = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'caption-side', {
        get: function () {
            var definition = require('./properties/captionSide').definition;
            Object.defineProperty(prototype, 'captionSide', definition);
            return this.captionSide;
        },
        set: function (v) {
            var definition = require('./properties/captionSide').definition;
            Object.defineProperty(prototype, 'captionSide', definition);
            this.captionSide = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'clear', {
        get: function () {
            var definition = require('./properties/clear').definition;
            Object.defineProperty(prototype, 'clear', definition);
            return this.clear;
        },
        set: function (v) {
            var definition = require('./properties/clear').definition;
            Object.defineProperty(prototype, 'clear', definition);
            this.clear = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'clip', {
        get: function () {
            var definition = require('./properties/clip').definition;
            Object.defineProperty(prototype, 'clip', definition);
            return this.clip;
        },
        set: function (v) {
            var definition = require('./properties/clip').definition;
            Object.defineProperty(prototype, 'clip', definition);
            this.clip = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'color', {
        get: function () {
            var definition = require('./properties/color').definition;
            Object.defineProperty(prototype, 'color', definition);
            return this.color;
        },
        set: function (v) {
            var definition = require('./properties/color').definition;
            Object.defineProperty(prototype, 'color', definition);
            this.color = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'colorInterpolation', {
        get: function () {
            var definition = require('./properties/colorInterpolation').definition;
            Object.defineProperty(prototype, 'colorInterpolation', definition);
            return this.colorInterpolation;
        },
        set: function (v) {
            var definition = require('./properties/colorInterpolation').definition;
            Object.defineProperty(prototype, 'colorInterpolation', definition);
            this.colorInterpolation = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'color-interpolation', {
        get: function () {
            var definition = require('./properties/colorInterpolation').definition;
            Object.defineProperty(prototype, 'colorInterpolation', definition);
            return this.colorInterpolation;
        },
        set: function (v) {
            var definition = require('./properties/colorInterpolation').definition;
            Object.defineProperty(prototype, 'colorInterpolation', definition);
            this.colorInterpolation = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'colorInterpolationFilters', {
        get: function () {
            var definition = require('./properties/colorInterpolationFilters').definition;
            Object.defineProperty(prototype, 'colorInterpolationFilters', definition);
            return this.colorInterpolationFilters;
        },
        set: function (v) {
            var definition = require('./properties/colorInterpolationFilters').definition;
            Object.defineProperty(prototype, 'colorInterpolationFilters', definition);
            this.colorInterpolationFilters = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'color-interpolation-filters', {
        get: function () {
            var definition = require('./properties/colorInterpolationFilters').definition;
            Object.defineProperty(prototype, 'colorInterpolationFilters', definition);
            return this.colorInterpolationFilters;
        },
        set: function (v) {
            var definition = require('./properties/colorInterpolationFilters').definition;
            Object.defineProperty(prototype, 'colorInterpolationFilters', definition);
            this.colorInterpolationFilters = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'colorProfile', {
        get: function () {
            var definition = require('./properties/colorProfile').definition;
            Object.defineProperty(prototype, 'colorProfile', definition);
            return this.colorProfile;
        },
        set: function (v) {
            var definition = require('./properties/colorProfile').definition;
            Object.defineProperty(prototype, 'colorProfile', definition);
            this.colorProfile = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'color-profile', {
        get: function () {
            var definition = require('./properties/colorProfile').definition;
            Object.defineProperty(prototype, 'colorProfile', definition);
            return this.colorProfile;
        },
        set: function (v) {
            var definition = require('./properties/colorProfile').definition;
            Object.defineProperty(prototype, 'colorProfile', definition);
            this.colorProfile = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'colorRendering', {
        get: function () {
            var definition = require('./properties/colorRendering').definition;
            Object.defineProperty(prototype, 'colorRendering', definition);
            return this.colorRendering;
        },
        set: function (v) {
            var definition = require('./properties/colorRendering').definition;
            Object.defineProperty(prototype, 'colorRendering', definition);
            this.colorRendering = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'color-rendering', {
        get: function () {
            var definition = require('./properties/colorRendering').definition;
            Object.defineProperty(prototype, 'colorRendering', definition);
            return this.colorRendering;
        },
        set: function (v) {
            var definition = require('./properties/colorRendering').definition;
            Object.defineProperty(prototype, 'colorRendering', definition);
            this.colorRendering = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'content', {
        get: function () {
            var definition = require('./properties/content').definition;
            Object.defineProperty(prototype, 'content', definition);
            return this.content;
        },
        set: function (v) {
            var definition = require('./properties/content').definition;
            Object.defineProperty(prototype, 'content', definition);
            this.content = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'counterIncrement', {
        get: function () {
            var definition = require('./properties/counterIncrement').definition;
            Object.defineProperty(prototype, 'counterIncrement', definition);
            return this.counterIncrement;
        },
        set: function (v) {
            var definition = require('./properties/counterIncrement').definition;
            Object.defineProperty(prototype, 'counterIncrement', definition);
            this.counterIncrement = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'counter-increment', {
        get: function () {
            var definition = require('./properties/counterIncrement').definition;
            Object.defineProperty(prototype, 'counterIncrement', definition);
            return this.counterIncrement;
        },
        set: function (v) {
            var definition = require('./properties/counterIncrement').definition;
            Object.defineProperty(prototype, 'counterIncrement', definition);
            this.counterIncrement = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'counterReset', {
        get: function () {
            var definition = require('./properties/counterReset').definition;
            Object.defineProperty(prototype, 'counterReset', definition);
            return this.counterReset;
        },
        set: function (v) {
            var definition = require('./properties/counterReset').definition;
            Object.defineProperty(prototype, 'counterReset', definition);
            this.counterReset = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'counter-reset', {
        get: function () {
            var definition = require('./properties/counterReset').definition;
            Object.defineProperty(prototype, 'counterReset', definition);
            return this.counterReset;
        },
        set: function (v) {
            var definition = require('./properties/counterReset').definition;
            Object.defineProperty(prototype, 'counterReset', definition);
            this.counterReset = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cssFloat', {
        get: function () {
            var definition = require('./properties/cssFloat').definition;
            Object.defineProperty(prototype, 'cssFloat', definition);
            return this.cssFloat;
        },
        set: function (v) {
            var definition = require('./properties/cssFloat').definition;
            Object.defineProperty(prototype, 'cssFloat', definition);
            this.cssFloat = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'css-float', {
        get: function () {
            var definition = require('./properties/cssFloat').definition;
            Object.defineProperty(prototype, 'cssFloat', definition);
            return this.cssFloat;
        },
        set: function (v) {
            var definition = require('./properties/cssFloat').definition;
            Object.defineProperty(prototype, 'cssFloat', definition);
            this.cssFloat = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cue', {
        get: function () {
            var definition = require('./properties/cue').definition;
            Object.defineProperty(prototype, 'cue', definition);
            return this.cue;
        },
        set: function (v) {
            var definition = require('./properties/cue').definition;
            Object.defineProperty(prototype, 'cue', definition);
            this.cue = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cueAfter', {
        get: function () {
            var definition = require('./properties/cueAfter').definition;
            Object.defineProperty(prototype, 'cueAfter', definition);
            return this.cueAfter;
        },
        set: function (v) {
            var definition = require('./properties/cueAfter').definition;
            Object.defineProperty(prototype, 'cueAfter', definition);
            this.cueAfter = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cue-after', {
        get: function () {
            var definition = require('./properties/cueAfter').definition;
            Object.defineProperty(prototype, 'cueAfter', definition);
            return this.cueAfter;
        },
        set: function (v) {
            var definition = require('./properties/cueAfter').definition;
            Object.defineProperty(prototype, 'cueAfter', definition);
            this.cueAfter = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cueBefore', {
        get: function () {
            var definition = require('./properties/cueBefore').definition;
            Object.defineProperty(prototype, 'cueBefore', definition);
            return this.cueBefore;
        },
        set: function (v) {
            var definition = require('./properties/cueBefore').definition;
            Object.defineProperty(prototype, 'cueBefore', definition);
            this.cueBefore = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cue-before', {
        get: function () {
            var definition = require('./properties/cueBefore').definition;
            Object.defineProperty(prototype, 'cueBefore', definition);
            return this.cueBefore;
        },
        set: function (v) {
            var definition = require('./properties/cueBefore').definition;
            Object.defineProperty(prototype, 'cueBefore', definition);
            this.cueBefore = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'cursor', {
        get: function () {
            var definition = require('./properties/cursor').definition;
            Object.defineProperty(prototype, 'cursor', definition);
            return this.cursor;
        },
        set: function (v) {
            var definition = require('./properties/cursor').definition;
            Object.defineProperty(prototype, 'cursor', definition);
            this.cursor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'direction', {
        get: function () {
            var definition = require('./properties/direction').definition;
            Object.defineProperty(prototype, 'direction', definition);
            return this.direction;
        },
        set: function (v) {
            var definition = require('./properties/direction').definition;
            Object.defineProperty(prototype, 'direction', definition);
            this.direction = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'display', {
        get: function () {
            var definition = require('./properties/display').definition;
            Object.defineProperty(prototype, 'display', definition);
            return this.display;
        },
        set: function (v) {
            var definition = require('./properties/display').definition;
            Object.defineProperty(prototype, 'display', definition);
            this.display = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'dominantBaseline', {
        get: function () {
            var definition = require('./properties/dominantBaseline').definition;
            Object.defineProperty(prototype, 'dominantBaseline', definition);
            return this.dominantBaseline;
        },
        set: function (v) {
            var definition = require('./properties/dominantBaseline').definition;
            Object.defineProperty(prototype, 'dominantBaseline', definition);
            this.dominantBaseline = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'dominant-baseline', {
        get: function () {
            var definition = require('./properties/dominantBaseline').definition;
            Object.defineProperty(prototype, 'dominantBaseline', definition);
            return this.dominantBaseline;
        },
        set: function (v) {
            var definition = require('./properties/dominantBaseline').definition;
            Object.defineProperty(prototype, 'dominantBaseline', definition);
            this.dominantBaseline = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'elevation', {
        get: function () {
            var definition = require('./properties/elevation').definition;
            Object.defineProperty(prototype, 'elevation', definition);
            return this.elevation;
        },
        set: function (v) {
            var definition = require('./properties/elevation').definition;
            Object.defineProperty(prototype, 'elevation', definition);
            this.elevation = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'emptyCells', {
        get: function () {
            var definition = require('./properties/emptyCells').definition;
            Object.defineProperty(prototype, 'emptyCells', definition);
            return this.emptyCells;
        },
        set: function (v) {
            var definition = require('./properties/emptyCells').definition;
            Object.defineProperty(prototype, 'emptyCells', definition);
            this.emptyCells = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'empty-cells', {
        get: function () {
            var definition = require('./properties/emptyCells').definition;
            Object.defineProperty(prototype, 'emptyCells', definition);
            return this.emptyCells;
        },
        set: function (v) {
            var definition = require('./properties/emptyCells').definition;
            Object.defineProperty(prototype, 'emptyCells', definition);
            this.emptyCells = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'enableBackground', {
        get: function () {
            var definition = require('./properties/enableBackground').definition;
            Object.defineProperty(prototype, 'enableBackground', definition);
            return this.enableBackground;
        },
        set: function (v) {
            var definition = require('./properties/enableBackground').definition;
            Object.defineProperty(prototype, 'enableBackground', definition);
            this.enableBackground = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'enable-background', {
        get: function () {
            var definition = require('./properties/enableBackground').definition;
            Object.defineProperty(prototype, 'enableBackground', definition);
            return this.enableBackground;
        },
        set: function (v) {
            var definition = require('./properties/enableBackground').definition;
            Object.defineProperty(prototype, 'enableBackground', definition);
            this.enableBackground = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fill', {
        get: function () {
            var definition = require('./properties/fill').definition;
            Object.defineProperty(prototype, 'fill', definition);
            return this.fill;
        },
        set: function (v) {
            var definition = require('./properties/fill').definition;
            Object.defineProperty(prototype, 'fill', definition);
            this.fill = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fillOpacity', {
        get: function () {
            var definition = require('./properties/fillOpacity').definition;
            Object.defineProperty(prototype, 'fillOpacity', definition);
            return this.fillOpacity;
        },
        set: function (v) {
            var definition = require('./properties/fillOpacity').definition;
            Object.defineProperty(prototype, 'fillOpacity', definition);
            this.fillOpacity = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fill-opacity', {
        get: function () {
            var definition = require('./properties/fillOpacity').definition;
            Object.defineProperty(prototype, 'fillOpacity', definition);
            return this.fillOpacity;
        },
        set: function (v) {
            var definition = require('./properties/fillOpacity').definition;
            Object.defineProperty(prototype, 'fillOpacity', definition);
            this.fillOpacity = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fillRule', {
        get: function () {
            var definition = require('./properties/fillRule').definition;
            Object.defineProperty(prototype, 'fillRule', definition);
            return this.fillRule;
        },
        set: function (v) {
            var definition = require('./properties/fillRule').definition;
            Object.defineProperty(prototype, 'fillRule', definition);
            this.fillRule = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fill-rule', {
        get: function () {
            var definition = require('./properties/fillRule').definition;
            Object.defineProperty(prototype, 'fillRule', definition);
            return this.fillRule;
        },
        set: function (v) {
            var definition = require('./properties/fillRule').definition;
            Object.defineProperty(prototype, 'fillRule', definition);
            this.fillRule = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'filter', {
        get: function () {
            var definition = require('./properties/filter').definition;
            Object.defineProperty(prototype, 'filter', definition);
            return this.filter;
        },
        set: function (v) {
            var definition = require('./properties/filter').definition;
            Object.defineProperty(prototype, 'filter', definition);
            this.filter = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'floodColor', {
        get: function () {
            var definition = require('./properties/floodColor').definition;
            Object.defineProperty(prototype, 'floodColor', definition);
            return this.floodColor;
        },
        set: function (v) {
            var definition = require('./properties/floodColor').definition;
            Object.defineProperty(prototype, 'floodColor', definition);
            this.floodColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'flood-color', {
        get: function () {
            var definition = require('./properties/floodColor').definition;
            Object.defineProperty(prototype, 'floodColor', definition);
            return this.floodColor;
        },
        set: function (v) {
            var definition = require('./properties/floodColor').definition;
            Object.defineProperty(prototype, 'floodColor', definition);
            this.floodColor = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'floodOpacity', {
        get: function () {
            var definition = require('./properties/floodOpacity').definition;
            Object.defineProperty(prototype, 'floodOpacity', definition);
            return this.floodOpacity;
        },
        set: function (v) {
            var definition = require('./properties/floodOpacity').definition;
            Object.defineProperty(prototype, 'floodOpacity', definition);
            this.floodOpacity = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'flood-opacity', {
        get: function () {
            var definition = require('./properties/floodOpacity').definition;
            Object.defineProperty(prototype, 'floodOpacity', definition);
            return this.floodOpacity;
        },
        set: function (v) {
            var definition = require('./properties/floodOpacity').definition;
            Object.defineProperty(prototype, 'floodOpacity', definition);
            this.floodOpacity = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font', {
        get: function () {
            var definition = require('./properties/font').definition;
            Object.defineProperty(prototype, 'font', definition);
            return this.font;
        },
        set: function (v) {
            var definition = require('./properties/font').definition;
            Object.defineProperty(prototype, 'font', definition);
            this.font = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontFamily', {
        get: function () {
            var definition = require('./properties/fontFamily').definition;
            Object.defineProperty(prototype, 'fontFamily', definition);
            return this.fontFamily;
        },
        set: function (v) {
            var definition = require('./properties/fontFamily').definition;
            Object.defineProperty(prototype, 'fontFamily', definition);
            this.fontFamily = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-family', {
        get: function () {
            var definition = require('./properties/fontFamily').definition;
            Object.defineProperty(prototype, 'fontFamily', definition);
            return this.fontFamily;
        },
        set: function (v) {
            var definition = require('./properties/fontFamily').definition;
            Object.defineProperty(prototype, 'fontFamily', definition);
            this.fontFamily = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontSize', {
        get: function () {
            var definition = require('./properties/fontSize').definition;
            Object.defineProperty(prototype, 'fontSize', definition);
            return this.fontSize;
        },
        set: function (v) {
            var definition = require('./properties/fontSize').definition;
            Object.defineProperty(prototype, 'fontSize', definition);
            this.fontSize = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-size', {
        get: function () {
            var definition = require('./properties/fontSize').definition;
            Object.defineProperty(prototype, 'fontSize', definition);
            return this.fontSize;
        },
        set: function (v) {
            var definition = require('./properties/fontSize').definition;
            Object.defineProperty(prototype, 'fontSize', definition);
            this.fontSize = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontSizeAdjust', {
        get: function () {
            var definition = require('./properties/fontSizeAdjust').definition;
            Object.defineProperty(prototype, 'fontSizeAdjust', definition);
            return this.fontSizeAdjust;
        },
        set: function (v) {
            var definition = require('./properties/fontSizeAdjust').definition;
            Object.defineProperty(prototype, 'fontSizeAdjust', definition);
            this.fontSizeAdjust = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-size-adjust', {
        get: function () {
            var definition = require('./properties/fontSizeAdjust').definition;
            Object.defineProperty(prototype, 'fontSizeAdjust', definition);
            return this.fontSizeAdjust;
        },
        set: function (v) {
            var definition = require('./properties/fontSizeAdjust').definition;
            Object.defineProperty(prototype, 'fontSizeAdjust', definition);
            this.fontSizeAdjust = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontStretch', {
        get: function () {
            var definition = require('./properties/fontStretch').definition;
            Object.defineProperty(prototype, 'fontStretch', definition);
            return this.fontStretch;
        },
        set: function (v) {
            var definition = require('./properties/fontStretch').definition;
            Object.defineProperty(prototype, 'fontStretch', definition);
            this.fontStretch = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-stretch', {
        get: function () {
            var definition = require('./properties/fontStretch').definition;
            Object.defineProperty(prototype, 'fontStretch', definition);
            return this.fontStretch;
        },
        set: function (v) {
            var definition = require('./properties/fontStretch').definition;
            Object.defineProperty(prototype, 'fontStretch', definition);
            this.fontStretch = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontStyle', {
        get: function () {
            var definition = require('./properties/fontStyle').definition;
            Object.defineProperty(prototype, 'fontStyle', definition);
            return this.fontStyle;
        },
        set: function (v) {
            var definition = require('./properties/fontStyle').definition;
            Object.defineProperty(prototype, 'fontStyle', definition);
            this.fontStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-style', {
        get: function () {
            var definition = require('./properties/fontStyle').definition;
            Object.defineProperty(prototype, 'fontStyle', definition);
            return this.fontStyle;
        },
        set: function (v) {
            var definition = require('./properties/fontStyle').definition;
            Object.defineProperty(prototype, 'fontStyle', definition);
            this.fontStyle = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontVariant', {
        get: function () {
            var definition = require('./properties/fontVariant').definition;
            Object.defineProperty(prototype, 'fontVariant', definition);
            return this.fontVariant;
        },
        set: function (v) {
            var definition = require('./properties/fontVariant').definition;
            Object.defineProperty(prototype, 'fontVariant', definition);
            this.fontVariant = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-variant', {
        get: function () {
            var definition = require('./properties/fontVariant').definition;
            Object.defineProperty(prototype, 'fontVariant', definition);
            return this.fontVariant;
        },
        set: function (v) {
            var definition = require('./properties/fontVariant').definition;
            Object.defineProperty(prototype, 'fontVariant', definition);
            this.fontVariant = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'fontWeight', {
        get: function () {
            var definition = require('./properties/fontWeight').definition;
            Object.defineProperty(prototype, 'fontWeight', definition);
            return this.fontWeight;
        },
        set: function (v) {
            var definition = require('./properties/fontWeight').definition;
            Object.defineProperty(prototype, 'fontWeight', definition);
            this.fontWeight = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'font-weight', {
        get: function () {
            var definition = require('./properties/fontWeight').definition;
            Object.defineProperty(prototype, 'fontWeight', definition);
            return this.fontWeight;
        },
        set: function (v) {
            var definition = require('./properties/fontWeight').definition;
            Object.defineProperty(prototype, 'fontWeight', definition);
            this.fontWeight = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'glyphOrientationHorizontal', {
        get: function () {
            var definition = require('./properties/glyphOrientationHorizontal').definition;
            Object.defineProperty(prototype, 'glyphOrientationHorizontal', definition);
            return this.glyphOrientationHorizontal;
        },
        set: function (v) {
            var definition = require('./properties/glyphOrientationHorizontal').definition;
            Object.defineProperty(prototype, 'glyphOrientationHorizontal', definition);
            this.glyphOrientationHorizontal = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'glyph-orientation-horizontal', {
        get: function () {
            var definition = require('./properties/glyphOrientationHorizontal').definition;
            Object.defineProperty(prototype, 'glyphOrientationHorizontal', definition);
            return this.glyphOrientationHorizontal;
        },
        set: function (v) {
            var definition = require('./properties/glyphOrientationHorizontal').definition;
            Object.defineProperty(prototype, 'glyphOrientationHorizontal', definition);
            this.glyphOrientationHorizontal = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'glyphOrientationVertical', {
        get: function () {
            var definition = require('./properties/glyphOrientationVertical').definition;
            Object.defineProperty(prototype, 'glyphOrientationVertical', definition);
            return this.glyphOrientationVertical;
        },
        set: function (v) {
            var definition = require('./properties/glyphOrientationVertical').definition;
            Object.defineProperty(prototype, 'glyphOrientationVertical', definition);
            this.glyphOrientationVertical = v;
        },
        enumerable: true,
        configurable: true
    });
    Object.defineProperty(prototype, 'glyph-orientation-vertical', {
        get: function () {
            var definition = require('./properties/glyphOrientationVertical').definition;
            Object.defineProperty(prototype, 'glyphOrientationVertical', definition);
            return this.glyphOrientationVertical;
